fishtank/src/cluster.test.ts

```diff
@@ -40,19 +40,110 @@ describe('Cluster', () => {
       const containerDatadir = join(
```

Comment:
This test creates a temporary directory that is not cleaned up. This may cause the test to pass even if the code is incorrect.

Consider the following situation:
1. Someone runs this test case. The directory with the `config.json` file is created but not removed.
2. The same person changes `cluster.ts` so that `config.json` is not written.
3. They run the test case again. The test passes because the `config.json` was leftover from the previous run, even though the code is incorrect.

Is this going to be addressed in a future PR?

Reply:
will remove the folder after tests